README.md

```diff
@@ -15,7 +15,8 @@ For this, simply run `./build.sh`.
 Next, for development, you would then run `./run-db.sh`.
 This creates a local database to use in development.
 If you would like to seed this database with production data,
-run `./download-prod.sh` and then `./upload-prod.sh`.
+run `./download-prod.sh` and then `./upload-prod.sh`. Note that before running './download-prod.sh' you must create a hidden file 'db.config' that configures the mysql server properly. 
```

Comment:
No worries about this this time, but in the future try to checkout `separate-api` again (will be `master` once a couple more days pass) and then make a new branch without these same changes.  It just helps us if maybe I were to reject that change but accept the other one.  That's just the purpose of makes branches / changes, etc. very specific.